app/assets/stylesheets/common.css

```diff
@@ -40,3 +40,8 @@ a:focus {
 .sul-embed-location-restricted-text {
   color: var(--stanford-digital-red);
 }
+
+.su-underline {
+  --underline-color: #ababa9;
```

Comment:
would you mind calling this variable `--stanford-40-black` to align with the component library?